src/home_robot_hw/home_robot_hw/nodes/goto_controller.py

```diff
@@ -145,6 +146,7 @@ def main(self):
         rospy.init_node("goto_controller")
 
         self.vel_command_pub = rospy.Publisher("stretch/cmd_vel", Twist, queue_size=1)
+        self.at_goal_pub = rospy.Publisher("stretch/at_goal", Bool, queue_size=1)
```

Comment:
Change to `goto_controller/at_goal` so it's more obvious that the goal in question is `goto_controller/goal`?